src/hosting/implicitInit.ts

```diff
@@ -54,8 +62,31 @@ export async function implicitInit(options: any): Promise<TemplateServerResponse
   }
 
   const configJson = JSON.stringify(config, null, 2);
+
+  const emulators: { [e in Emulators]?: Address } = {};
+  for (const e of EMULATORS_SUPPORTED_BY_USE_EMULATOR) {
+    const info = EmulatorRegistry.getInfo(e);
+
+    if (info) {
+      // IPv6 hosts need to be quoted using brackets.
+      const host = info.host.includes(":") ? `[${info.host}]` : info.host;
+      emulators[e] = {
+        host,
+        port: info.port,
+      };
+    }
+  }
+  const emulatorsJson = JSON.stringify(emulators, null, 2);
+
+  const js = INIT_TEMPLATE
```

Comment:
Please run the formatter.

Reply:
Wow the formatter makes this awful haha but I gotta stan prettier